symfony/twig-bundle/6.4/config/packages/twig.yaml

```diff
@@ -0,0 +1,7 @@
+twig:
+    default_path: '%kernel.project_dir%/templates'
```

Comment:
isn't this the default already in TwigBundle ? If yes, I suggest removing the explicit config duplicating it, to simplify things.

Reply:
yes, it's already the case

Reply:
yes, it's already the case

Reply:
I'll remove this config: https://github.com/symfony/symfony/blob/b0f5d7e7db2a0b1ff38fb2c8c933cb8edd05d07b/src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php#L145C33-L145C33

Reply:
We have a few settings that are just the defaults to show how to override things. This is one of them. Maybe not that frequent, I just wanted to highlight why it was there.